### PR TITLE
added two parameters to cloudsuite_data_serving benchmark

### PR DESCRIPTION
**Pull request by ustiugov**

### CODE REVIEW

File: perfkitbenchmarker/linux_benchmarks/cloudsuite_data_serving_benchmark.py [hunk 1]
```diff
@@ -28,11 +28,23 @@
...
+                     'Operation count to be executed.',
+                     lower_bound=1)
+
 BENCHMARK_NAME = 'cloudsuite_data_serving'
 BENCHMARK_CONFIG = """
 cloudsuite_data_serving:
   description: >
       Run YCSB client against Cassandra servers.
+      Specify record count and operation count with
+      --rec_count and --op_count.
```
Comment (hildrum) [inline perfkitbenchmarker/linux_benchmarks/cloudsuite_data_serving_benchmark.py]: Shouldn't this be --cloudsuite_data_serving_rec_count and not rec_count?

Comment (ustiugov) [inline perfkitbenchmarker/linux_benchmarks/cloudsuite_data_serving_benchmark.py]: thank you, fixed

**Comment by hildrum** [comment]
LGTM.  Let me know if you're ready to merge.

**Comment by ustiugov** [comment]
ok, thank you. I am ready

**Pull request merged**